apcd-cms/src/taccsite_cms/urls_custom.py

```diff
@@ -1,3 +1,6 @@
 from django.urls import path, include
 
-custom_urls = []
+custom_urls = [
+    path('custom_example/', include('apps.custom_example.urls', namespace='custom_example')),
+    path('apcd/', include('apps.submission_form.urls', namespace='apcd'))
```

Comment:
**Required. (But untested.)** So that the URL of the page is as expected (i.e. as it was for original form).

```suggestion
    path('register/request-to-submit/', include('apps.submission_form.urls', namespace='apcd'))
```